modules/wp-rest/classes/elementor-settings.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Elementor\Modules\WpRest\Classes;
+
+use Exception;
+use WP_REST_Request;
+use WP_REST_Server;
+use WP_REST_Response;
+
+defined( 'ABSPATH' ) || exit;
+
+class ElementorSettings {
+
+	public function register(): void {
```

Comment:
Just a question, we do have an abstraction to interface with the WP REST API (example):

https://github.com/elementor/elementor/blob/main/modules/checklist/data/controller.php

But I see our developers all using their own implementations. (Here and Global Classes for example).

TBH, we did find that base classes confusing to work with, is this the reason you did not use it as well?

Reply:
Ok